### ISSUE TITLE
modification of names with special character

### ISSUE THREAD

**Issue by sebdalgarno** [ateucher/rmapshaper#91]
hi @ateucher I'm not sure if this behaviour is intended, but it caught me a bit off guard. 

```
library(sf)
library(rmapshaper)
nc <- st_read(system.file(package = "sf", "shape/nc.shp"))
nc[["new name"]] <- 1
nc <- nc[, "new name"]

tmp <- ms_simplify(nc)

> names(nc)
[1] "new name" "geometry"
> names(tmp)
[1] "new.name" "geometry"
> 
```

Is it possible to leave the variable names as they were even if they have special characters?

**Comment by ateucher** [comment]
I think it should be possible, thanks for reporting this.

**Comment by ateucher** [comment]
This should be fixed in ongoing work in #118
